hydra/test_utils/ax_sweeper_plugin.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+from typing import Any
+
+from omegaconf import DictConfig
+
+log = logging.getLogger(__name__)
+
+
+def banana(cfg: DictConfig) -> Any:
```

Comment:
What is this doing here?
The tested can be in the plugin.
When we have additional similar plugins we can consider moving common logic to a central place.

Reply:
Right :)